src/main/java/animals/controller/Game.java

```diff
@@ -0,0 +1,162 @@
+package animals.controller;
+
+import animals.localization.LanguageRule;
+import animals.localization.LanguagesRules_en;
+import animals.localization.LanguagesRules_eo;
+import animals.model.Node;
+import animals.service.FileService;
+import animals.service.KnowledgeTreeService;
+import animals.service.MenuService;
+
+import java.time.LocalTime;
+import java.util.ResourceBundle;
+import java.util.Scanner;
+
+import static animals.service.ResourceBundleService.getLocalString;
+
+public class Game {
+    public static final String FILE_LOCATION_NAME = getFileLocationName();
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final LocalTime MORNING_START = LocalTime.of(5, 1);
+    private static final LocalTime MORNING_END = LocalTime.of(12, 0);
+    private static final LocalTime DAY_START = LocalTime.of(12, 1);
+    private static final LocalTime DAY_END = LocalTime.of(18, 0);
+
+    FileService fs = new FileService();
+    KnowledgeTreeService kts = new KnowledgeTreeService();
+    LanguageRule lr = getLanguageRule();
+
+    private LanguageRule getLanguageRule() {
+        return System.getProperty("user.language").equals("eo") ? new LanguagesRules_eo() : new LanguagesRules_en();
+    }
+
+    public void run(String type) {
+        String FILE_NAME = FILE_LOCATION_NAME + type;
+        System.out.println(checkTimeOfDay() + "\n");
+
+        var exist = fs.checkFile(FILE_NAME);
```

Comment:
What if file exists and is not empty, but it is just `{}`?
I think we should try to load data and then check if a note is valid